daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	Read() (io.Reader, error)
```

Comment:
Yes, makes sense. We should take a look how it can be implemented for other drivers. Take a look at journald - it's my first candidate for `docker logs`.